book/src/simple-payment-and-state-verification.md

```diff
@@ -0,0 +1,180 @@
+# Simple Payment and State Verification
+
+The transaction contains a signature. Validators store those signatures for a
+short period of time to ensure the transaction is not processed more than once.
+Validators provide a JSON RPC endpoint, which clients can use to query the
+cluster if a transaction has been recently processed. Validators also provide a
+PubSub notification, whereby a client registers to be notified when a given
+signature is observed by the validator. While these two mechanisms allow a
+client to verify a payment, they are not a proof and rely on a trusted
+validator.
+
+## Light Clients
+
+Light clients are cluster participants that are not running a fullnode.  The
+goal of this proposal is to provide a level of security that is greater than
+trusting a remote fullnode without requiring the light client to spend a lot of
+resources on verifying the ledger.
+
+The additional level of security for a light client depends on an initial
+canonical set of validators the light client considers to be the stakeholders of
+the cluster. As that set is changed, the client can update its internal set of
+known validators with [receipts](#receipts). This may become challenging with a
+large number of delegated stakes.
+
+Fullnodes themselves may want to use light client apis for performance reasons.
+For example, during the initial launch of a fullnode, the fullnode may use a
+cluster provided checkpoint of the state and verify it with a receipt.
+
+## Receipts
+
+The basic idea for receipts is to create a minimal hash chain from the
+transaction to a slice of PoH ledger that proves that the desired set of
+validators have voted on a block contain that transaction to the desired
```

Comment:
"the desired set of validators has voted on a block containing that transaction"
Also, I do not understand the clause "to the desired confirmation". Do you mean "with the desired confirmation"?

Reply:
Do it!

Reply:
Thinking of replacing this whole paragraph with

```
A receipt is a minimal proof that a transaction has been included in a block,
that the block has been voted on by the client's preferred set of validators and
that the votes have reached the desired confirmation.
```

What do you think?

Reply:
Way better!